apps/web/app/[locale]/admin/submissions/EventTable.tsx

```diff
@@ -56,38 +56,63 @@ export default async ({ searchParams, params }: SubmissionsTableProps) => {
     updatedAt: string;
   }>;
 
-  const sqlQueryParams: (string | boolean)[] = [];
+  const sqlQueryParams: (string | boolean | number)[] = [];
   let paramIndex = 1;
-  if (queryParams.search) {
+  if (search) {
     dataQuery += ` AND (
       (flow_data ->> 'govIEEmail') ILIKE $${paramIndex}
       OR (flow_data ->> 'myGovIdEmail') ILIKE $${paramIndex}
       OR (flow_data ->> 'firstName') ILIKE $${paramIndex}
       OR (flow_data ->> 'lastName') ILIKE $${paramIndex}
       )`;
     paramIndex++;
-    sqlQueryParams.push(`%${queryParams.search}%`);
+    sqlQueryParams.push(`%${search}%`);
   }
 
-  const filters = queryParams.filters;
-  for (const [key, value] of Object.entries(filters)) {
-    dataQuery += ` AND (flow_data ->> $${paramIndex} = $${paramIndex + 1})`;
-    sqlQueryParams.push(key);
-    sqlQueryParams.push(value);
-    paramIndex += 2;
+  if (filters) {
+    for (const [key, value] of Object.entries(filters)) {
+      dataQuery += ` AND (flow_data ->> $${paramIndex} = $${paramIndex + 1})`;
+      sqlQueryParams.push(key);
+      sqlQueryParams.push(value);
```

Comment:
really minor, just a matter of style: you could push both entries in a single call
```suggestion
      sqlQueryParams.push(key, value);
```